WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+final class TitleAndSubtitleTableViewCell: UITableViewCell {
```

Comment:
nit: how about including the status UI in the cell name? something like `TitleAndSubitlteAndStatusTableViewCell` so that it's easier to discover from search in the future 😃

Reply:
I've renamed this in a7f3064fabee7ae8d9a1e8a241435b6190fcdba0 👍